### DISCUSSION TITLE
Reply only mentions, ignoring the replies of my original tweet

### DISCUSSION THREAD

**Discussion by oGabrielArruda** [tweepy/tweepy#1577] [Questions] [answered]
I'm making a bot that answer all mentions. For find those mentions, I'm using this code:

` tweets = api.mentions_timeline(last_seen_id, tweet_mode='extended')`

The problem is: if I've already answered a mention and a random user answer this tweet, or answer the tweet that my bot was mentioned, the API consider it a mention.

I want to ignore all the replies of my tweet and the replies of the tweet that I was mentioned, and consider just the first tweet as a mention.

Is there an API method that I can use to do this? Or do I have to create a boolean function like 'is_not_reply(tweet)'?

**Comment by Harmon758** [comment, marked as answer]
You'll have to use one of the `in_reply_to_status_id`, `in_reply_to_status_id_str`, `in_reply_to_user_id`, `in_reply_to_user_id_str`, or `in_reply_to_screen_name` attributes of each Status/[Tweet object](https://developer.twitter.com/en/docs/twitter-api/v1/data-dictionary/object-model/tweet) and filter out replies yourself.
